src/components/forms/Form.js

```diff
@@ -436,7 +436,7 @@ class _Form extends Component {
 
         dispatch(showModal(
           <BlockComponent nextLocation={nextLocation} unblock={unblock} />,
-          { isUnclosable: true }
+          { isUnclosable: true, extraClassName:'confirm-change-' +  _this2.props.name }
```

Comment:
this.props ici :) quand on travaille sur le code src

Reply:
et ca serait meme mieux de faire
```
extraClassName: `confirm-change-${this.props.name}`
```

Reply:
du coup c'est bon